### PR TITLE
docs/conf: cleanup

### PR DESCRIPTION
**Pull request by umarcor**
The Sphinx configuration was created in 2018 and the comments were not updated. This PR removes them and instead adds a reference to the updated page in the Sphinx documentation.

### CODE REVIEW

**Review by kgugala — APPROVED** [review]
LGTM

File: docs/source/conf.py [hunk 1]
```diff
@@ -1,105 +1,48 @@
...
+# -- General configuration -------------------------------------------------------------------------
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+project = 'symbiflow-arch-defs'
+copyright = '2018, Various'
+author = 'Various'
+
+# The short X.Y version.
```
Comment (mithro) [inline docs/source/conf.py:17]: Can you add a TODO to pull these from `git-describe`?

**Review by mithro — APPROVED** [review]
Two minor comments.

Merge when fixed without waiting for more feedback.

File: docs/source/conf.py (hunk 1 above)
Comment (umarcor) [reply docs/source/conf.py]: Done in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/2397/commits/0c0843195de232cd157b8bfb52bab5e2120e207b.

File: docs/source/conf.py [hunk 2]
```diff
@@ -1,105 +1,48 @@
...
 
-# -- General configuration ------------------------------------------------
+# -- General configuration -------------------------------------------------------------------------
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+project = 'symbiflow-arch-defs'
+copyright = '2018, Various'
+author = 'Various'
```
[the remark this answers is not in the document]
Comment (umarcor) [reply docs/source/conf.py:15]: This repo seems not to have consistent/updated headers. Shall I use the one from https://github.com/SymbiFlow/symbiflow-examples/pull/249/files ?

**Pull request merged**